### PR TITLE
Test timing of SA pull secret creation

### PR DESCRIPTION
**Pull request by adambkaplan**
Ensure service account pull secrets are created in 2 seconds or less.

### CODE REVIEW

File: test/extended/controller_manager/pull_secret.go [hunk 1]
```diff
@@ -195,4 +195,20 @@ var _ = g.Describe("[sig-devex][Feature:OpenShiftControllerManager]", func() {
...
+
+		g.By("creating test service account")
+		sa := &corev1.ServiceAccount{
+			ObjectMeta: metav1.ObjectMeta{Name: "new-sa", Namespace: saNamespace},
+		}
+
+		sa, err := clusterAdminKubeClient.CoreV1().ServiceAccounts(sa.Namespace).Create(sa)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		g.By("waiting for service account's pull secret to be created")
+		_, _, err = waitForServiceAccountPullSecret(clusterAdminKubeClient, sa.Namespace, sa.Name, 5, time.Second)
```
Comment (adambkaplan) [inline test/extended/controller_manager/pull_secret.go]: @mfojtik @smarterclayton Is 5 seconds a reasonable maximum to fail on?

File: test/extended/controller_manager/pull_secret.go [hunk 2]
```diff
@@ -195,4 +195,20 @@ var _ = g.Describe("[sig-devex][Feature:OpenShiftControllerManager]", func() {
 			t.Fatalf("error waiting for pull secret deletion: %v", err)
 		}
 	})
+
+	g.It("create service account pull secrets timing", func() {
```
Comment (bparees) [inline test/extended/controller_manager/pull_secret.go:199]:
```suggestion
	g.It("checks service account pull secret creation timing", func() {
```

**Comment by bparees** [comment]
@adambkaplan please squash and let me know when you've seen 4-5 passing runs and i'll lgtm.

**Comment by adambkaplan** [comment]
/retest

hit by CI issues

**Comment by danehans** [comment]
@adambkaplan does this PR need to be linked to [BZ 1819849](https://bugzilla.redhat.com/show_bug.cgi?id=1819849)?

**Comment by adambkaplan** [comment]
/close

No longer relevant

**Pull request closed without merging**